common/ini-file.cpp

```diff
@@ -36,6 +36,7 @@ bool INIFile::isValidName(const String &name) {
 }
 
 INIFile::INIFile() {
+	_8bitMode = false;
```

Comment:
The name is a bit confusing. It could be renamed to something like _allowNonEnglishCharacters